projects/packages/masterbar/src/admin-menu/class-atomic-admin-menu.php

```diff
@@ -429,8 +429,16 @@ public function add_options_menu() {
 			);
 		}
 
-		// @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
-		add_submenu_page( 'options-general.php', esc_attr__( 'Hosting Configuration', 'jetpack-masterbar' ), __( 'Hosting Configuration', 'jetpack-masterbar' ), 'manage_options', 'https://wordpress.com/hosting-config/' . $this->domain, null, 11 );
+		$has_feature_atomic = wpcom_site_has_feature( \WPCOM_Features::ATOMIC );
```

Comment:
Hmm, probably I'm not understanding this well, but we're already inside `AtomicAdminMenu` class, what exactly is this line checking 🤔

Reply:
I have a site like below

![image](https://github.com/user-attachments/assets/c6ce46c9-7d81-4907-937b-aa0552c4a196)

When I go to Hosting Configuration, it becomes

![image](https://github.com/user-attachments/assets/bc500e6f-dd8e-4fc3-8d55-c69375045289)

I'm unsure whether it's a real case or not 🤔 But this line is to check where the atomic feature is still available.

Reply:
Oh I see.... OK then 😅